package.json

```diff
@@ -10,9 +10,10 @@
     "test:watch": "jest --watch",
     "lint": "eslint .",
     "format": "prettier --write {config,src,test}/**/*.js",
-    "ci": "yarn format && yarn lint && yarn test",
+    "ci": "yarn format && yarn lint && yarn test -i",
```

Comment:
Why should we run them in band in CI?

Reply:
Travis runs much slower with parallel Jest 😢